src/cljs/zetawar/events/ui.cljs

```diff
@@ -210,6 +213,21 @@
                   :action/r r}]
                 [::clear-selection]]}))
 
+(defmethod router/handle-event ::repair-targeted
+  [{:as handler-ctx :keys [db]} _]
+  (let [game (app/current-game db)
+        cur-faction-color (game/current-faction-color game)
+        [repairer-q repairer-r] (app/selected-hex db)
+        [wounded-q  wounded-r]  (app/targeted-hex db)]
```

Comment:
Understandable. I tried to find a pair of words with the same length to match "attacker/defender," but couldn't decide on anything.